COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
-file for the given project.
+* Adds both `debug` and `release` frameworks as output binaries for all macOS, iOS, tvOS, and watchOS targets.
+* Creates a `podspec` task which generates a [Podspec](https://guides.cocoapods.org/syntax/podspec.html)
+file for the project.
 
-The podspec generated includes a path to an output framework and script phases which automate building
-this framework during a build process of an Xcode project. Some fields of the podspec file can be
-configured using the `kotlin.cocoapods { ... }` code block.
+The `Podspec` file includes a path to an output framework and script phases which automate building
+this framework during a build process of an Xcode project. 
 
-<div class="sample" markdown="1" theme="idea" mode="kotlin">
+## Add dependencies on Pod libraries
 
-```kotlin
-// Apply plugins.
-plugins {
-    id("org.jetbrains.kotlin.multiplatform") version "1.3.30"
-    id("org.jetbrains.kotlin.native.cocoapods") version "1.3.30"
-}
-
-// CocoaPods requires the podspec to have a version.
-version = "1.0"
-
-kotlin {
-    cocoapods {
-        // Configure fields required by CocoaPods.
-        summary = "Some description for a Kotlin/Native module"
-        homepage = "Link to a Kotlin/Native module homepage"
-
-        // The name of the produced framework can be changed.
-        // The name of the Gradle project is used here by default.
-        frameworkName = "my_framework"
+You can add dependencies between a Kotlin project and Pod libraries [stored in the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
+and [stored locally](#add-a-dependency-on-a-pod-library-stored-locally).
+
+[Complete initial configuration](#install-the-cocoapods-dependency-manager-and-plugin), and when you add a new dependency, 
+just re-import the project in IntelliJ IDEA. The new dependency will be added automatically. No additional 
+steps are required.
+
+### Add a dependency on a Pod library from the CocoaPods repository
+
+1. Add dependencies on a Pod library that you want to use from the CocoaPods repository with `pod()`  to `build.gradle.kts` 
+(`build.gradle`) of your project.  
+    > You can also add dependencies on subspecs.
+    {:.note}                                                                                                                                                              >
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            pod("AFNetworking", "~> 4.0.0")
+            
+            pod("SDWebImage/MapKit")
+        }
     }
-}
+    ```
+    
+    </div>
+
+2. Re-import the project.
+
+To use these dependencies from Kotlin code, import packages `cocoapods.<library-name>`.
```

Comment:
To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.

Reply:
✅